src/Keboola/ConfigMigrationTool/Migration/GenericCopyMigration.php

```diff
@@ -13,18 +13,22 @@
 class GenericCopyMigration extends DockerAppMigration
 {
 
+    public const SKIP_ENCRYPTION = true;
+    public const FORCE_ENCRYPTION = false;
```

Comment:
to mi prijde dost ujety definovat true a false jako konstanty https://thedailywtf.com/articles/What_Is_Truth_0x3f_

Reply:
tak bys tam dal čísla? chtěl jsem místo true/false v argumentu použít konstantu.

Reply:
ne, dal bych tam true/false nic mensiho uz neexistuje. 

Ale kdyby ses chtel objevit na dailywtf, tak tam muzes dat string `true` a `false`, ulozit je do kolekce a do konstanty si zapsat jejich index treba :)